src/main/java/test.java

```diff
@@ -0,0 +1,11 @@
+import browser.Driver;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+
+public class test {
+    @Test
+    public void startDriver(){
```

Comment:
startDriver is not part of the test, this should be the precondition for all test
@BeforeClass or as @Before

Please review